injector/src/main/java/edu/ucr/cs/riple/injector/Helper.java

```diff
@@ -395,16 +395,22 @@ public static String getPackageName(String name) {
   }
 
   /**
-   * Corrects Path starting with prefix: {@code file}
+   * Deserializes a Path instance from a string.
    *
-   * @param value Path to file.
+   * @param path Path to file.
    * @return The modified Path.
    */
-  public static String extractPath(String value) {
-    if (!new File(value).exists() && value.startsWith("file:")) {
-      return value.substring("file:".length());
+  public static Path deserializePath(String path) {
+    if (path.startsWith("file://")) {
```

Comment:
We should probably also test for any other URI scheme and throw an error, no? What happens with Jar URIs here?

Reply:
Added case for jar paths: 2ae07c2f80a6756de35bd5095b15027f05c323bc